frontend/src/metabase/components/Logs.jsx

```diff
@@ -11,8 +11,8 @@ import "react-ansi-style/inject-css";
 import _ from "underscore";
 
 export default class Logs extends Component {
-    constructor(props, context) {
-        super(props, context);
+    constructor() {
```

Comment:
@tlrobinson I tend to like omitting it if we don't actually need to reference it so that the presence of props acts as a quick way to see whether the constructor is doing anything interesting I need to be aware of. I'd liken it to when in stateless components we use destructuring to only pull out the named props we actually use. But that being said I don't have a strong preference.